dandiapi/api/views/asset.py

```diff
@@ -82,6 +83,12 @@ class AssetViewSet(DetailSerializerMixin, GenericViewSet):
     filter_backends = [filters.DjangoFilterBackend]
     filterset_class = AssetFilter
 
+    def get_throttles(self) -> list[BaseThrottle]:
+        if self.action == 'list':
+            throttles = [*self.throttle_classes, AnonRateThrottle]
+            return [throttle() for throttle in throttles]
+        return super().get_throttles()
```

Comment:
Last suggestion I promise. I realized that we're basically reimplementing the base [`get_throttles`](https://github.com/encode/django-rest-framework/blob/4f10c4e43ee57f4a2e387e0c8d44d28d21a3621c/rest_framework/views.py#L280) method. So it may be more concise to just do the following:
```suggestion
    def get_throttles(self) -> list[BaseThrottle]:
        throttles = self.throttle_classes
        if self.action == 'list':
            throttles.append(AnonRateThrottle)
        return [throttle() for throttle in throttles]
```

Reply:
Doesn't this also modify the `self.throttle_classes` variable itself? Assigning it to `throttles` is not making a copy of the list, but just assigning a reference to it to `throttles`. So calling `.append` on `throttles` will modify the instance variable too, right?

Reply:
Good catch. I modified the suggestion to include a `.copy`.